sumatra/parameters.py

```diff
@@ -130,6 +130,17 @@ def _dict_diff(a, b):
         return result1, result2
 
 
+def flatten_dict(init, lkey=''):
```

Comment:
The `parameters` package, which is one of Sumatra's dependencies, already contains a function `nesteddictflatten()` which does the same thing as this, as far as I can see, so you can just import that directly in `web/views.py`.